lib/color.js

```diff
@@ -219,6 +218,75 @@ class ColorXY {
         return kelvinToMireds(kelvin);
     }
 
+    /**
+     * Converts CIE color space to RGB color space
+     * From: https://github.com/usolved/cie-rgb-converter/blob/master/cie_rgb_converter.js
+     * @return {ColorRGB}
+     */
+    toRGB() {
```

Comment:
@sjorge continuing on:

> xy->rgb function https://github.com/usolved/cie-rgb-converter/blob/master/cie_rgb_converter.js seems to apply reverse gamma correction, not sure this should be applied or not, we get small deviations if we do so I skipped this in ColorXY.toRGB().


I think we should, we also do it when setting rgb via xy.

Reply:
I remove the comment so it gets applies the reverse correction..